library/src/main/java/com/getstream/sdk/chat/view/messageinput/AttachmentsController.kt

```diff
@@ -249,23 +196,20 @@ internal class AttachmentsController(
             val attachments = withContext(dispatchersProvider.ioDispatcher) {
                 storageHelper.getAttachmentsFromUriList(view.context, uriList)
             }
-            attachments.forEach { selectAttachment(it, false) }
+            attachments.forEach { selectAttachment(it) }
```

Comment:
For what reason do you do it iteratively instead of adding all files once? It's not so good from a performance point of view. I'd like to create a separate method for select one attachment (actually it works only for media mode) and add all attachments (adding all list to selected ones by one invocation)  from the platform picker.
It again divides code for two different modes (media/ non-media) that makes code cleaner.